3rdParty/iresearch/core/formats/columnstore2.cpp

```diff
@@ -368,6 +360,62 @@ class column_base : public column_reader,
     return *stream_;
   }
 
+ protected:
+  column_header& mutable_header() { return hdr_; }
+  void reset_stream(const index_input* stream) { stream_ = stream; }
+
+  bool allocate_buffered_memory(size_t size) {
+    assert(size < static_cast<size_t>(std::numeric_limits<int64_t>::max()));
+    if (buffered_memory_accounter_ &&
```

Comment:
I suggest we set default noop buffered_memory_accounter_ in case if nothing is set